templates/squid.conf.header.erb

```diff
@@ -17,7 +17,11 @@ cache_replacement_policy      <%= @cache_replacement_policy %>
 memory_replacement_policy     <%= @memory_replacement_policy %>
 <% end -%>
 
-<% if @logformat -%>
+<% if @logformat.is_a?(Array) %>
+<% @logformat.each do |logformat_line| -%>
+logformat                     <%= logformat_line %>
+<% end -%>
+<% else -%>
 logformat                     <%= @logformat %>
 <% end -%>
```

Comment:
Maybe it's worth avoiding the duplication, something like:

```erb
 <% [@logformat].flatten.each do |logformat| -%>
logformat                     <%= logformat %>
<% end -%>
```

Reply:
Never saw that format before, sgtm, pushed up.